src/functions/keycloak/remove-user-attribute/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "delete",
+        path: "/users/{id}/attributes/{attributeName?}",
```

Comment:
I think {attributeName} is fine, so the ? can be removed or is it there for a reason?